clients/richclient/src/main/java/org/openecard/richclient/RichClient.java

```diff
@@ -492,4 +503,61 @@ private static Long getRegInt(WinReg.HKEY hk, String key, String value, @Nullabl
 	return defaultValue;
     }
 
+    private static void setJnaRuntimeDiretory() throws IOException {
```

Comment:
We should move this to a seperate file, there is already too much code in the RichClient class.

Reply:
The Jna Runtime fix has been moved to its own class (though I'm not sure about the name of the class yet).